### PR TITLE
Feature form validation

### PR DESCRIPTION
**Pull request by Obote**
### What does this repo do?

- When the user submits the form, you check if the email is in lower case.
- If the validation is OK, the form is sent.
- If the validation is not OK, you show an error message to the user informing them of the error and the form is not sent.

### CODE REVIEW

File: Javascript/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+//     tech: ['HTML', 'JavaScript', 'Css', 'Ruby'],
+//     imgUrl: './assets/images/bg.jpg',
+//   },
+//   {
+//     title: 'About the project #4',
+//     description: 'Description of the project, date and languages',
+//     tech: ['HTML', 'JavaScript', 'Css', 'Ruby'],
+//     imgUrl: './assets/images/bg.jpg',
+//   },
+// ];
```
Comment (bellom) [inline Javascript/index.js]: Kindly delete commented code, it would be professional to make use of git history instead.

File: index.html [hunk 2]
```diff
@@ -217,12 +219,12 @@ <h3 id="contact">CONTACT US</h3>
...
+            <form action="https://formspree.io/f/xdoylybq" method="POST" id="desktopForm">
                 <div class="form-box">
                     <input class="input1" type="text" id="firstName" name="name" placeholder="First name" required>
-                    <input class="input2" type="text" id="lasNname" name="name" placeholder="Last name" required>
-                </div>
+                    <input class="input2" type="text" id="lasNname" name="name" placeholder="Last name" required>                </div>
                 <input type="email" id="mail1" name="_replyto" placeholder="jeremy@gmail.com"  maxlength="500" required>
+                <small id="lowerCase1" class="mailErorr formShow">Please write the email in lower case</small>
                 <textarea name="message" id="msg1" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
                 <button class="button" type="submit">See Project</button>
```
Comment (bellom) [inline index.html]: I think you might want to change this to text `submit`.

File: index.html [hunk 3]
```diff
@@ -217,12 +219,12 @@ <h3 id="contact">CONTACT US</h3>
...
             <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </P>
-            <form action="https://formspree.io/f/xdoylybq" method="POST">
+            <form action="https://formspree.io/f/xdoylybq" method="POST" id="desktopForm">
                 <div class="form-box">
                     <input class="input1" type="text" id="firstName" name="name" placeholder="First name" required>
-                    <input class="input2" type="text" id="lasNname" name="name" placeholder="Last name" required>
-                </div>
+                    <input class="input2" type="text" id="lasNname" name="name" placeholder="Last name" required>                </div>
                 <input type="email" id="mail1" name="_replyto" placeholder="jeremy@gmail.com"  maxlength="500" required>
+                <small id="lowerCase1" class="mailErorr formShow">Please write the email in lower case</small>
```
Comment (bellom) [inline index.html]: Let's show the error message near the submit button.

image

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Obote  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

_Kindly check the comments under the review._

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Javascript/index.js (hunk 1 above)
Comment (rloterh) [reply Javascript/index.js]: This issue was well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Obote , 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 :shipit: .
Congratulations! 🎉 🥂


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**